datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -1380,24 +1380,16 @@ mod tests {
                 "+---+-----+-----------------+",
                 "| a | b   | COUNT(1)[count] |",
                 "+---+-----+-----------------+",
-                "|   | 1.0 | 1               |",
```

Comment:
The behaviour of my branch is consistent with the one that run in slt.
Not sure which one is the expected behaviour 🤔

Reply:
When in doubt, I always run in postgres to see what it says

Reply:
I have checked both postgres and duckdb, the `spill` one is the expected result. But why the `non-spill` version has the different result